client/components/nux-notice/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
```

Comment:
We can codemod it all at once when we upgrade to 16. `wp-calypso` will face the same transition so we'll be able to borrow its migration script from them.